tools/policy-gen/bootstrap/root.go

```diff
@@ -157,6 +167,7 @@ package kuma.plugins.policies.{{ .nameLower }}.{{ .version }};
 import "mesh/options.proto";
 option go_package = "{{ .module }}/{{.nameLower}}/api/{{ .version }}";
 
+import "common/v1alpha1/targetref.proto";
```

Comment:
If you want to do it right you need to add a `{{- if .generateTargetRef || .generate.To || .generateFrom}}`